lib/pharos/phases/base.rb

```diff
@@ -8,8 +8,7 @@ module Phases
     class Base
       # @return [String]
       def self.title
-        # XXX: prettier
-        self.name
+        self.const_get(:PHASE_TITLE) || self.name
```

Comment:
Could be:

```ruby
def self.title(title = nil)
  @title ||= title || name
end

class FooPhase < Base
  title "Configure foofoo"

  def call
    ...
  end
end
```

Reply:
I originally tried `self.title=(title)`, but it seems like `title = "foo"` doesn't invoke that, I guess it defines a const instead?

Is it a good pattern to use the same method as both an accessor and setter as well?